### PR TITLE
examples and test

### PR DESCRIPTION
**Pull request by sudiptoguha**
*Issue #, if available:* 258

*Description of changes:* fixes the commented out test and adds examples.

### CODE REVIEW

**Comment by jotok** [comment]
Can we add all the examples to the Main example class? It's a command-line runner that makes it easy to invoke any of the examples.

https://github.com/aws/random-cut-forest-by-aws/blob/main/Java/examples/src/main/java/com/amazon/randomcutforest/examples/Main.java#L35-L40

File: Java/examples/src/main/java/com/amazon/randomcutforest/examples/dynamicdensity/DynamicDensity.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    }
+
+    @Override
+    public String command() {
+        return "dynamic_sampling";
+    }
+
+    @Override
+    public String description() {
+        return "check dynamic sampling";
```
Comment (jotok) [inline Java/examples/src/main/java/com/amazon/randomcutforest/examples/dynamicdensity/DynamicDensity.java:42]: Can we be any more descriptive?

File: Java/examples/src/main/java/com/amazon/randomcutforest/examples/dynamicnearneighbor/DynamicNearNeighbor.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    }
+
+    @Override
+    public String command() {
+        return "dynamic_near_neighbor";
+    }
+
+    @Override
+    public String description() {
+        return "check dynamic near neighbor";
```
Comment (jotok) [inline Java/examples/src/main/java/com/amazon/randomcutforest/examples/dynamicnearneighbor/DynamicNearNeighbor.java:41]: Same.

File: Java/examples/src/main/java/com/amazon/randomcutforest/examples/dynamicdensity/DynamicDensity.java (hunk 1 above)
Comment (sudiptoguha) [reply Java/examples/src/main/java/com/amazon/randomcutforest/examples/dynamicdensity/DynamicDensity.java]: :) fixed.

File: Java/examples/src/main/java/com/amazon/randomcutforest/examples/dynamicnearneighbor/DynamicNearNeighbor.java (hunk 2 above)
Comment (sudiptoguha) [reply Java/examples/src/main/java/com/amazon/randomcutforest/examples/dynamicnearneighbor/DynamicNearNeighbor.java]: fixed.

**Comment by sudiptoguha** [comment]
added examples to main

**Review by jotok — APPROVED** [review]

**Pull request merged**